readme.md

```diff
@@ -20,10 +20,10 @@ library.
 
 ## [API Documentation](https://docs.rs/bincode/)
 
-## Bincode in the wild
+## Bincode in the Wild
```

Comment:
I don't think the word `wild` needs to be capitalized

Reply:
That's just to be consistent with the other first level headers in the Readme.